pkg/stub/handler.go

```diff
@@ -60,6 +61,12 @@ func NewHandler() sdk.Handler {
 	h.skippedImagestreams = make(map[string]bool)
 	h.skippedTemplates = make(map[string]bool)
 
+	h.defaultResourceTimer = time.NewTimer(5 * time.Second)
+	go func() {
+		<-h.defaultResourceTimer.C
+		h.CreateDefaultResourceIfNeeded()
+	}()
```

Comment:
@legionus this is a pattern you might want to copy in your operator.. basically a short grace period in case someone is trying to create their own resource object before the operator creates the default one.